bin/ff_map

```diff
@@ -66,9 +66,6 @@ def main():
                                     args.tpr,
                                     args.traj,
                                     in_memory=True)
-    if args.pbc_complete:
-        print("INFO - PBC completeing trajectory")
-        init_universe.pbc_complete()
```

Comment:
I think it would still be a good idea from within the Universe handler to tell we are pbc completing and also propagate the pbc complete flag to leave the option to not pbc complete.

Reply:
I think taking periodicity into account is the most natural thing and in this form doesn't need special mention (nor turning off)